### DISCUSSION TITLE
PR by dockerfile

### DISCUSSION THREAD

**Discussion by ademar59** [renovatebot/renovate#13302] [Request Help] [answered]
I have a monorepo dedicated for dockerfiles with a lot of dockerfiles.
I want to create PR by dockerfile without listing all the files (easy to add new dockerfile).

I don't find any configuration for that

**Comment by viceice** [comment]
Checkout additional branch prefix and template values

**Reply by viceice** [reply]
https://docs.renovatebot.com/templates/

**Reply by ademar59** [reply]
I don't understand how branch prefix could help me

I have a project with:
- folder1
  - dockerfile
- folder2
  - dockerfile
- folder3
  - dockerfile 
- folder4
  - dockerfile
 ...
 
 I don't want to create a lot of packageRules with all the paths parameter

**Reply by rarkins** [reply]
Use templating, Eg additionalBranchPrefix={{parentDir}}

Search this repo for past discussions/examples of these terms

**Comment by ademar59** [comment, marked as answer]
Thanks it works like a charm

I have added this to have a pretty commit message:
`"commitMessagePrefix": "chore({{parentDir}}): dependencies"`

![image](https://user-images.githubusercontent.com/8971839/147737814-eee3f4ab-f5bb-4ad1-bd47-689ea8e48107.png)
